lib/rules/headers/dl.js

```diff
@@ -192,9 +192,12 @@ exports.check = function (sr, done) {
 
     if (dts.Implementation) {
         var linkImple = dts.Implementation.dd.querySelector("a") && dts.Implementation.dd.querySelector("a").href;
+        if (!linkImple) sr.error(self, "implelink-not-found");
         if (linkImple && !linkImple.trim().toLowerCase().startsWith('https://')) {
             sr.error(self, "implelink-should-be-https", {link: linkImple});
         }
+    } else {
+        sr.error(self, "implelink-not-found");
```

Comment:
Implementation reports are required only for CR, PR and REC. This would trigger an error for the other types of document